core/store/orm/config_test.go

```diff
@@ -56,17 +56,17 @@ func TestConfig_Profiles(t *testing.T) {
 		{"mainnet", "1", 500000, 1000000000000000000},
 		{"kovan", "42", 500000, 1000000000000000000},
 
-		{"optimism", "10", 500000, 1000000000000000000},
-		{"optimism", "69", 500000, 1000000000000000000},
-		{"optimism", "420", 500000, 1000000000000000000},
+		{"optimism", "10", 500000, 100000000000000},
+		{"optimism", "69", 500000, 100000000000000},
+		{"optimism", "420", 500000, 100000000000000},
 
-		{"bscMainnet", "56", 500000, 1000000000000000000},
-		{"hecoMainnet", "128", 500000, 1000000000000000000},
-		{"fantomMainnet", "250", 500000, 3500000000000000},
-		{"fantomTestnet", "4002", 500000, 3500000000000000},
-		{"polygonMatic", "800001", 500000, 1000000000000000000},
+		{"bscMainnet", "56", 500000, 100000000000000},
+		{"hecoMainnet", "128", 500000, 100000000000000},
+		{"fantomMainnet", "250", 500000, 100000000000000},
```

Comment:
Looks like you undid the change for fantom?

Reply:
This was intentional - it turned out that Integrations misunderstood what this parameter did, and thought it was something else.

The correct values are:

1 LINK for mainnet
0.0001 LINK everywhere else